docs/_integration-manager/camel-routes/camel-history-management.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Camel File Processing History Management
+nav_order: 5
+parent: File Folder Listener
+---
+# Camel File Processing History Management
+All processed & unprocessed file history can be magnaged here.
+
+
+## Manage Processed File History
+All unprocessed files will be moved to this specified configured location.
+
+Example:
+```
+<route id="manage-processed-file-history" autoStartup="false">
+        <from uri="file://{{sharedDataPath}}/listener/processed?recursive=true&amp;filterFile=${file:modified} &lt; ${date:now-14d}"/>
+        <to uri="log:trash?level=OFF"/>
+</route>
+```
+
+
+## Manage Unprocessed File History
+
+All unprocessed files will be moved to this specified configured location.
+
+Example:
+```
+<route id="manage-unprocessed-file-history" autoStartup="false">
```

Comment:
Not sure, whether we need to specify these individual properties we have used in this configurations. 
Example - "filterFile" - Why this filter file has been used ? or anything ? Any suggestions?

Reply:
https://camel.apache.org/components/4.0.x/file-component.html#_endpoint_query_option_filterFile

https://stackoverflow.com/questions/47532169/camel-filterfile-dsl-example

you should explain how filterFile setting works here, namely the camel simple language syntax, e.g. ${file:modified} &lt; ${date:now-14d} means the file was modified over 14 days ago